### PR TITLE
change to new contributor list format

### PR DESCRIPTION
**Pull request by ThatGravyBoat**

### CODE REVIEW

**Comment by qtlunya** [comment]
Can I have `:3` too? 🥺

(Also, my IGN changed from AlexiaLuna to owoLuna.)

**Comment by j10a1n15** [comment]
> Can I have `:3` too? 🥺

I want it too plsss

File: constants/ContributorList.json [hunk 1]
```diff
@@ -25,12 +25,146 @@
...
+    },
+    "Potat_owo": {
+      "spinny": true
+    },
+    "Pnda__": {
+      "spinny": true
+    },
+    "StopUsingSBE": {
+      "spinny": true
+    }
```
Comment (hannibal002) [inline constants/ContributorList.json:168]: who are  those?

File: constants/ContributorList.json [hunk 2]
```diff
@@ -25,12 +25,146 @@
...
     "Skirtwearer",
     "Skyblock_Lobby",
+    "ThatGravyBoat",
     "thisiscad",
     "Throwpo",
     "Thunderblade73",
     "Vahvl",
     "Vixid",
     "Wambux",
+    "Wolfie586",
```
Comment (hannibal002) [inline constants/ContributorList.json:35]: who is that?

**Comment by ItsEmpa** [comment]
me too if possible 🥺

**Review by hannibal002 — CHANGES_REQUESTED** [review]
Can we remove all entries in contributors where `"suffix": "§c:O"`, and then just use this as default if not in usernames?

also having names like Biscut under a "contributor" list is misleading.

File: constants/ContributorList.json (hunk 2 above)
Comment (CalMWolfs) [reply constants/ContributorList.json:35]: my alt

File: constants/ContributorList.json (hunk 1 above)
Comment (CalMWolfs) [reply constants/ContributorList.json:168]: sba people

File: constants/Contributors.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        "Mikecraft1224": {},
+        "NetheriteMiner": {},
+        "Obsidianninja11": {},
+        "owoLuna": {
+            "suffix": "§d:3"
+        },
+        "oxsss": {},
+        "ReyMaratov": {},
+        "Skirtwearer": {
+            "suffix": "§c:3"
```
Comment (j10a1n15) [inline constants/Contributors.json:37]:
```suggestion
            "suffix": "§d:3"
```
Sorry for this

**Comment by Fix3dll** [comment]
@ThatGravyBoat IGN Fix3dll

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**